### ISSUE TITLE
Message groups with a randomizer

### ISSUE THREAD

**Issue by Road-block** [mcd8604/wowmessagebarker#27]
This is an enhancement request.

My main usage for this addon would be to replace recruitment ad snippets.
At the moment I am using in-game notepads with the ability to run Lua snippets.
A typical construct I would use is have a couple or three variations in an array and use `local msg = msgArray[random(1,#msgArray)]` before sending.

So what this would entail is moving the text content of the message object 1 level in and have the addon randomize the output.
Or as a further refinement assign weights to each message so the "business" message goes out 70% of the time an the "humor" 30%.

Anyway that's one use case for pitching my enhancement idea 😄 

TL;DR: Text content of the message object moved one level deep into an array to make message permutations possible.

**Comment by Road-block** [comment]
I can try pitch it as a pull request if that helps.
